R/zchunk_L102.GDP.R

```diff
@@ -181,14 +184,13 @@ module_socioeconomics_L102.GDP <- function(command, ...) {
       standardize_iso('ISO') %>%
       change_iso_code('rou', 'rom') %>%
       gather(year, gdp.rate, -iso) %>%
-      full_join( gdp_mil90usd_ctry %>% select(iso) %>% unique,
-                 by = 'iso') %>%
+      full_join(gdp_mil90usd_ctry %>% select(iso) %>% unique, by = 'iso') %>%
       mutate(gdp.rate = if_else(gdp.rate == 'n/a', '0', gdp.rate)) %>% # Treat string 'n/a' as missing.
       mutate(year = as.integer(year),
              gdp.rate = as.numeric(gdp.rate)) %>%
-      replace_na(list(year=2010)) %>% # have to do this for `complete` to work as expected.
+      replace_na(list(year = 2010)) %>% # have to do this for `complete` to work as expected.
```

Comment:
Do we want to do something more flexible here than hard-coding 2010?

(I know this is unrelated to your changes, just flagging it)